t/rnacentral.t

```diff
@@ -0,0 +1,48 @@
+use Test::More;
+
```

Comment:
No strict, and you can use Alessandro's handy test defaults module to guarantee configuration is consistent:
`use FindBin qw/$Bin/;
use lib "$Bin";
use TestDefaults;`